main.go

```diff
@@ -48,22 +53,36 @@ func main() {
 		return
 	}
 
-	if opts.NArgs() != 1 && opts.NArgs() != 2 {
+	if opts.NArgs() > 2 {
 		getopt.PrintUsage(os.Stderr)
 		os.Exit(1)
 		return
 	}
 
+	var target *url.URL
+	var err error
 	args := opts.Args()
-	target, err := url.Parse(args[0])
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to parse target '%s': %s\n", args[0], err)
-		os.Exit(1)
-		return
+	if len(args) == 0 {
+		target, err = url.Parse(os.Getenv("GOTCHA_BACKEND"))
```

Comment:
What happens here if we run the command without arguments, but _don't_ specify the `GOTCHA_BACKEND` environment variable?

I would like to see a more helpful error message like:
> No backend host specified, and no $GOTCHA_BACKEND environment variable set
> If you are deploying gotcha as a Cloud Foundry application, don't forget to `cf set-env appname GOTCHA_BACKEND https://host/url

That way, when I forget how to use this, the tools tell me what I done forgot.